### PR TITLE
Add packaging files and config for goreleaser

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: build/post-install.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    printf "\033[32m Post Install of an clean install\033[0m\n"
+    printf "\033[32m Reload the service unit from disk\033[0m\n"
+    systemctl daemon-reload ||:
+    printf "\033[32m Unmask the service\033[0m\n"
+    systemctl unmask webhook-go.service ||:
+    printf "\033[32m Set the preset flag for the service unit\033[0m\n"
+    systemctl preset webhook-go.service ||:
+    printf "\033[32m Set the enabled flag for the service unit\033[0m\n"
+    systemctl enable webhook-go.service ||:
+    systemctl restart webhook-go.service ||:
```
Comment (bastelfreak) [inline build/post-install.sh:13]: do we have to autostart the service? debian does this but I am not a big fan of it. also without a user-provided config it doesnt make sense to start the service?

Comment (dhollinger) [reply build/post-install.sh:13]: It's pretty standard for packages to auto-start the service. 

I don't like it either, but I also don't want to confuse users by changing expected behavior

Comment (bastelfreak) [reply build/post-install.sh:13]: Is that common anywhere except for Debian? RHEL, Gentoo and  Arch Linux dont enable/start services after the installation.

Comment (dhollinger) [reply build/post-install.sh]: Updated with logs on what to do instead of auto-starting

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**